hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/profile/WriteProfiles.java

```diff
@@ -150,8 +150,8 @@ public static void clean(String path) {
       Path basePath,
       HoodieInstant instant,
       HoodieTimeline timeline) {
-    byte[] data = timeline.getInstantDetails(instant).get();
     try {
+      byte[] data = timeline.getInstantDetails(instant).get();
       return Option.of(HoodieCommitMetadata.fromBytes(data, HoodieCommitMetadata.class));
     } catch (IOException e) {
       // make this fail safe.
```

Comment:
IMO, here we should at least log the detailed exception information and stack trace. WDYT?

Reply:
I think so, I will modify it.